packages/ubuntu_bootstrap/lib/pages/source/applications_selection_page.dart

```diff
@@ -10,12 +10,10 @@ import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru/yaru.dart';
 import 'package:yaru_widgets/yaru_widgets.dart';
 
-export 'source_model.dart' show kFullSourceId, kMinimalSourceId;
-
-/// A page where the user can decide whether they want to install 3rd party
-/// drivers or codecs.
-class SourcePage extends ConsumerWidget with ProvisioningPage {
-  SourcePage({super.key});
+/// A page where the user can decide whether they want to do a full installation
+/// with all selected applications or a minimal installation.
+class ApplicationsSelectionPage extends ConsumerWidget with ProvisioningPage {
+  ApplicationsSelectionPage({super.key});
```

Comment:
"Source" is such a bad name though, it can refer to so many things :smile:
What about making it clearer in the dartdocs that it is using source in subiquity?

Reply:
Especially when we expose it in the config file

Reply:
Of course, "source" is much more general than what we need in the UI of the desktop installer, but it probably makes sense in the broader scope of the server installer. "Applications" might fit better with the current design and default source catalog, but I think it's too specific, since those things can change frequently. So I'd rather stick with a name that's consistent with the underlying business logic :)

Reply:
Ah, now I understand what you mean. I just ran the TPM test again and in there it lists all kinds of options, could those appear to the normal user too?

Reply:
Fixed in https://github.com/canonical/ubuntu-desktop-provision/pull/438/commits/e34aea406bd06e6a9c478e01b456028f2fb0692d

Reply:
Those in particular are just test cases that the user won't see. I don't know if there'll be different options in the source catalog for OEMs or if they'll be the standard desktop ones.